nevergrad/benchmark/test_experiments.py

```diff
@@ -21,7 +21,7 @@ def test_experiments_registry(name: str, maker: Callable[[], Iterator[experiment
     with patch("shutil.which", return_value="here"):  # do not check for missing packages
         with datasets.mocked_data():  # mock mlda data that should be downloaded
             check_maker(maker)  # this is to extract the function for reuse if other external packages need it
-        if name not in {"mlda", "mldaas", "realworld", "powersystems", "powersystemsbig"}:
+        if name not in {"mlda", "mldaas", "realworld", "powersystems", "powersystemsbig", "fastgames"}:
```

Comment:
eventually we need to solve this

Reply:
actually tests pass for "fastgames" so I remove it from that list